tasks/lammps/env.py

```diff
@@ -13,9 +10,26 @@
 DOCKER_LAMMPS_DIR = join(DOCKER_PROJ_ROOT, "third-party", "lammps")
 DOCKER_NATIVE_INSTALL_DIR = join(DOCKER_PROJ_ROOT, "build", "native-install")
 DOCKER_LAMMPS_BINARY = join(DOCKER_NATIVE_INSTALL_DIR, "bin", "lmp")
-DOCKER_LAMMPS_DATA_FILE = join(
-    DOCKER_LAMMPS_DIR, "examples", "controller", "in.controller.wall"
-)
 
 LAMMPS_FAASM_USER = "lammps"
 LAMMPS_FAASM_FUNC = "main"
+LAMMPS_FAASM_DATA_PREFIX = "/lammps-data"
+
+# Define the different benchmarks we run in LAMMPS
+
+BENCHMARKS = {
+    "lj": {"data": ["bench/in.lj"], "out_file": "compute"},
+    "compute": {"data": ["bench/in.lj"], "out_file": "compute"},
+    "controller": {
+        "data": ["examples/controller/in.controller.wall"],
+        "out_file": "network",
+    },
+    "network": {
+        "data": ["examples/controller/in.controller.wall"],
+        "out_file": "network",
+    },
+    "eam": {"data": ["bench/in.eam", "bench/Cu_u3.eam"], "out_file": "eam"},
+    "chute": ["bench/in.chute", "bench/data.chute"],
+    "rhodo": ["bench/in.rhodo", "bench/data.rhodo"],
+    "chain": ["bench/in.chain", "bench/data.chain"],
```

Comment:
Not sure why these last three benchmarks have a different format to the other four. Should these entries be `"data"`? If so, it would be good to make them a `"data"` entry and perhaps add an empty `"out_file"`.

Reply:
Good catch, I forgot to update this ones, thanks.